src/Controller/Admin/AnswerCrudController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\Answer;
+use Doctrine\ORM\EntityManagerInterface;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Action;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Actions;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Crud;
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+use EasyCorp\Bundle\EasyAdminBundle\Field\ChoiceField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\TextareaField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\TextField;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+
+class AnswerCrudController extends AbstractCrudController
+{
+    public static function getEntityFqcn(): string
+    {
+        return Answer::class;
+    }
+
+    public function updateEntity(EntityManagerInterface $entityManager, $entityInstance): void
+    {
+        parent::updateEntity($entityManager, $entityInstance); // TODO: Change the autogenerated stub
```

Comment:
Remember to remove todos after they have been done.
```suggestion
        parent::updateEntity($entityManager, $entityInstance);
```